test_hessian/src/main/java/test/TestCustomReply.java

```diff
@@ -115,4 +116,274 @@ public void customReplyUntypedVariableListHasNull() throws Exception {
         }
         output.flush();
     }
+
+    public void customReplyTypedFixedList_A0() throws Exception {
```

Comment:
`TypedFixedList` may be simpified as 

```java
public void customReplyTypedFixedList_A0() throws Exception {
    A0[] o = new A0[]{new A0(), new A0(), null};
    output.writeObject(o);
    output.flush();
}
```

because java decode array as `TypedFixedList` defaultly.

Reply:
Yes, I will change.